packages/web/components/earn/position/index.tsx

```diff
@@ -121,55 +130,55 @@ export const EarnPosition = () => {
           </div>
         </div>
         <EarnPositionChartSection />
-      </div>
+      </div> */}
     </div>
   );
 };
 
-const EarnPositionChartSection = () => {
-  const { t } = useTranslation();
+// const EarnPositionChartSection = () => {
+//   const { t } = useTranslation();
 
-  return (
-    <div className="relative mt-5 flex h-52">
-      <div className="absolute flex gap-1">
-        <ChartButton
-          label={t("tokenInfos.chart.xDay", {
-            d: "1",
-          })}
-          onClick={() => {}}
-          selected={false}
-        />
-        <ChartButton
-          label={t("tokenInfos.chart.xDay", {
-            d: "7",
-          })}
-          onClick={() => {}}
-          selected={false}
-        />
-        <ChartButton label={"1M"} onClick={() => {}} selected={true} />
-        <ChartButton
-          label={t("tokenInfos.chart.xYear", {
-            y: "1",
-          })}
-          onClick={() => {}}
-          selected={false}
-        />
-        <ChartButton
-          label={t("tokenInfos.chart.all", {
-            y: "1",
-          })}
-          onClick={() => {}}
-          selected={false}
-        />
-      </div>
-      <TokenPairHistoricalChart
-        minimal
-        showTooltip
-        showGradient
-        annotations={[]}
-        data={mockData}
-        domain={[0.6228583428571428, 0.691070457]}
-      />
-    </div>
-  );
-};
+//   return (
```

Comment:
why is this commented out?

Reply:
i commented it out because at the moment the chart won't be used as the data is not available. Would you prefer if i delete it entirely?

Reply:
no that's fine 👍